src/cloud-cli/cli.ts

```
@@ -16,8 +16,7 @@ import { credentialsExist } from "./utils";
 
 const program = new Command();
 
-const DEFAULT_HOST = "localhost"
-const DEFAULT_PORT = "8080"
+const DEFAULT_HOST = process.env.DBOS_DOMAIN; // TODO: Once we have a "production" cluster, hardcode its domain name here
```

We could panic early here if `DEFAULT_HOST` ends up `undefined`